README.md

```diff
@@ -23,6 +23,9 @@ You can configure the _iExec Core Scheduler_ with the following properties:
 | `IEXEC_CORE_PORT` | Server port of the _iExec Core Scheduler_. | Positive integer | `13000` |
 | `MONGO_HOST` | _MongoDB_ server host. Cannot be set with URI. | String | `localhost` |
 | `MONGO_PORT` | _MongoDB_ server port. Cannot be set with URI. | Positive integer | `13002` |
+| `IEXEC_PLATFORM_REGISTRY` | _iExec Platform Registry_ server URL. | URL | none |
+| `IEXEC_PLATFORM_REGISTRY_STACK` | [optional] Use a specific "profile" of the _iExec Platform Registry_ targeting specific iExec services you want to connect to. | String | none |
```

Comment:
```suggestion
| `IEXEC_PLATFORM_REGISTRY_STACK` | [optional] Use a specific "profile" of the _iExec Platform Registry_ targeting specific iExec services you want to connect to. Value depends on what is offered by the remote registry, it can be `bellecour-production` or any custom profile exposed by the registry. | String | none |
```

Reply:
@mcornaton Please merge this one if it answers your question :pray:

Reply:
When reading this line, I'm still unsure of what is expected there. How are profiles declared in the registry? Which value should this variable match?
It may be easier if we take an example. [In our registry repo](https://github.com/iExecBlockchainComputing/iexec-platform-services-config/blob/iexec-core-integration/application-65535.yml), we have a `main` branch with no code and a `iexec-core-integration-tests` branch with an `application-65535.yml` file. Is the "profile" `65535`? Does that mean "profile" is here used as the Spring concept? That is quite unclear for me.

Reply:
Here is a sample:
```
/{application}/{profile}[/{label}]
/{application}-{profile}.yml
/{label}/{application}-{profile}.yml
/{application}-{profile}.properties
/{label}/{application}-{profile}.properties
```
https://docs.spring.io/spring-cloud-config/docs/current/reference/html/#_spring_cloud_config_server

Does it answer your question?
Profile depend on how you configuration server is configured, it can be from native files or git

Reply:
Hope you prefer without profile wording https://github.com/iExecBlockchainComputing/iexec-core/pull/546/commits/e10ee56cf4e2addac2a6b61b6be0be58f5171845

Reply:
Sure, thanks!